projects/packages/videopress/src/client/block-editor/blocks/video/components/privacy-and-rating-panel/privacy-and-rating-settings.native.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import { PanelBody, SelectControl, ToggleControl } from '@wordpress/components';
+import { Platform } from '@wordpress/element';
+import { __, _x } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import {
+	VIDEO_PRIVACY_LEVELS,
+	VIDEO_PRIVACY_LEVEL_PRIVATE,
+	VIDEO_PRIVACY_LEVEL_PUBLIC,
+	VIDEO_PRIVACY_LEVEL_SITE_DEFAULT,
+	VIDEO_RATING_G,
+	VIDEO_RATING_PG_13,
+	VIDEO_RATING_R_17,
+} from '../../../../../state/constants';
+
+/**
+ * React component that renders the settings within the privacy and ratings panel.
+ *
+ * @param {object} props - Component props.
+ * @param {object} props.attributes - Block attributes.
+ * @param {Function} props.setAttributes - Function to set block attributes.
+ * @param {boolean} props.privateEnabledForSite - True if the site's privacy is set to Private.
+ * @returns {import('react').ReactElement} - Settings to change video's privacy and ratings.
+ */
+export default function PrivacyAndRatingSettings( {
+	attributes,
+	setAttributes,
+	privateEnabledForSite,
+} ) {
+	const { privacySetting, rating, allowDownload, displayEmbed } = attributes;
+
+	const privacyLabels = {
+		private: _x( 'Site Default (Private)', 'VideoPress privacy setting', 'jetpack-videopress-pkg' ),
+		public: _x( 'Site Default (Public)', 'VideoPress privacy setting', 'jetpack-videopress-pkg' ),
+	};
+
+	const privacyOptionSiteDefault = {
+		value: String( VIDEO_PRIVACY_LEVELS.indexOf( VIDEO_PRIVACY_LEVEL_SITE_DEFAULT ) ),
+		label: privateEnabledForSite ? privacyLabels.private : privacyLabels.public,
+	};
+	const privacyOptionPublic = {
+		value: String( VIDEO_PRIVACY_LEVELS.indexOf( VIDEO_PRIVACY_LEVEL_PUBLIC ) ),
+		label: _x( 'Public', 'VideoPress privacy setting', 'jetpack-videopress-pkg' ),
+	};
+	const privacyOptionPrivate = {
+		value: String( VIDEO_PRIVACY_LEVELS.indexOf( VIDEO_PRIVACY_LEVEL_PRIVATE ) ),
+		label: _x( 'Private', 'VideoPress privacy setting', 'jetpack-videopress-pkg' ),
+	};
+
+	return (
+		<PanelBody
+			title={ Platform.isWeb && __( 'Privacy and rating', 'jetpack-videopress-pkg' ) }
```

Comment:
We won't need this check in either the web or native files following this change. 

In addition, this separate `<PrivacyAndRatingSettings/>` component was created in https://github.com/Automattic/jetpack/pull/29477/commits/43b954ff643a88b63980780bc31b337fc9f1823f in order to share as much functionality between the web and native as possible. As the reasoning behind that commit is now somewhat redundant, what do you think about reverting back to having all of the code for the Privacy and Rating component in the `index` files? I could see a case for keeping this more modular and don't feel overly strongly, but also don't think there's a strong need for this to be a separate component now. Happy to go with whatever your preference is here.

Reply:
> We won't need this check in either the web or native files following this change.

Good catch, I'll remove it for the native file.

> In addition, this separate <PrivacyAndRatingSettings/> component was created in https://github.com/Automattic/jetpack/commit/43b954ff643a88b63980780bc31b337fc9f1823f in order to share as much functionality between the web and native as possible. As the reasoning behind that commit is now somewhat redundant, what do you think about reverting back to having all of the code for the Privacy and Rating component in the index files? I could see a case for keeping this more modular and don't feel overly strongly, but also don't think there's a strong need for this to be a separate component now. Happy to go with whatever your preference is here.

@SiobhyB I totally agree, now that we need to separate `PrivacyAndRatingSettings` component into two files we can revert the change introduced in https://github.com/Automattic/jetpack/commit/43b954ff643a88b63980780bc31b337fc9f1823f. This PR is subject to get file conflicts as it's updating most of the native files, so if possible, I'd like to propose merging it as is and applying the revert in a separate PR. WDYT?

Reply:
> > We won't need this check in either the web or native files following this change.
> 
> Good catch, I'll remove it for the native file.

Fix applied in https://github.com/Automattic/jetpack/pull/29894/commits/e518e8a877d9a4cc6651939219a5fc40caa9639b.

Reply:
> I'd like to propose merging it as is and applying the revert in a separate PR. WDYT?

@fluiddot, that makes sense to me, and apologies I'd originally missed [your comment](https://github.com/Automattic/jetpack/pull/29894/commits/e518e8a877d9a4cc6651939219a5fc40caa9639b#r1157005332) stating that intention. Thanks for removing the platform-specific condition, too!